src/diffusers/pipelines/wuerstchen/__init__.py

```diff
@@ -0,0 +1,8 @@
+from ...utils import is_torch_available, is_transformers_available
+
+
+if is_transformers_available() and is_torch_available():
+    from .modules import DiffNeXt, EfficientNetEncoder, Prior
```

Comment:
We should not have a file called `modules` every component should have their own file